docs/samples/Microsoft.ML.Samples/Dynamic/ProjectionTransforms.cs

```diff
@@ -55,7 +55,7 @@ public static void Example()
             //0.165 0.117 -0.547  0.014
 
             // A pipeline to project Features column into L-p normalized vector.
-            var lpNormalizePipeline = ml.Transforms.LpNormalize(nameof(SamplesUtils.DatasetUtils.SampleVectorOfNumbersData.Features), normKind: Transforms.LpNormalizingEstimatorBase.NormFunction.L1);
+            var lpNormalizePipeline = ml.Transforms.LpNormalize(nameof(SamplesUtils.DatasetUtils.SampleVectorOfNumbersData.Features), norm: Transforms.LpNormalizingEstimatorBase.NormFunction.L1);
```

Comment:
>norm [](start = 134, length = 4)

We should be using long-form names here.

Reply:
Ok. `LpNormalize` will be replaced by `LpNormNormalize`.

---
In reply to: [264826528](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264826528) [](ancestors = 264826528)